### PR TITLE
1.2 quizlet importing

### PR DESCRIPTION
**Pull request by Gabe0214**
Users can now copy their exported deck from Quizlet and paste them directly into the mNeme platform.

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -46,11 +47,11 @@ firebaseAppId = process.env.REACT_APP_APP_ID;
 // }
 
 const config = {
-  apiKey: firebaseApiKey,
```
Comment (GCJ2) [inline frontend/src/App.js]: These need to be as environmental variables.

**Review by GCJ2 — CHANGES_REQUESTED** [review]
Update code to use environmental variables.

**Review by GCJ2 — APPROVED** [review]
Thanks for updating this.

**Review by reeceap124 — COMMENTED** [review]
Looking good!

**Review by reeceap124 — APPROVED** [review]

**Pull request merged**